docs/release-notes/changelog.rst

```diff
@@ -3,6 +3,193 @@
+        :type: bugfix
+        :pr: 3285
+        :issue: 3201
+
+        Fix a bug that would prevent default values for dataclasses and ``msgspec.Struct`` s to be included in the
+        OpenAPI schema.
+
+    .. change:: Pydantic v2 error handling/serialization when for non-Pydantic exceptions
+        :type: bugfix
+        :pr: 3286
+        :issue: 2365
+
+        Fix a bug that would cause a :class:`TypeError` when non-Pydantic errors are raised during Pydantic's
+        validation process while using DTOs.
+
+    .. change:: Fix OpenAPI schema generation for paths with path parameters of different types on the same path
+        :type: bugfix
+        :pr: 3293
+        :issue: 2700
+
+        Fix a bug that would cause no OpenAPI schema to be generated for paths with path
+        parameters that only differ on the path parameter type, such as ``/{param:int}``
+        and ``/{param:str}``. This was caused by an internal representation issue in
+        Litestar's routing system.
+
+    .. change:: Document unconsumed path parameters
+        :type: bugfix
+        :pr: 3295
+        :issue: 3290
+
+        Fix a bug where path parameters not consumed by route handlers would not be included in the OpenAPI schema.
+
+        This could would not include the ``{param}`` in the schema, yet it is still required to be passed
+        when calling the path.
+
+    .. change:: Allow for console output to be silenced
+        :type: feature
+        :pr: 3180
+
+        This change introduces optional environment variables that allow customizing the "Application" name displayed
+        in the console output and suppressing the initial ``from_env`` or the ``Rich`` info table at startup.
+
+        This provides flexibility in tailoring the console output to better integrate Litestar into larger applications
+        or CLIs.
+
+    .. change:: Add flash plugin
+        :type: feature
+        :pr: 3145
+        :issue: 1455
+
+        Added a flash plugin akin to Django or Flask that uses the request state
+
+    .. change:: Use memoized :paramref:`~.handlers.HTTPRouteHandler.request_class` and :paramref:`~.handlers.HTTPRouteHandler.response_class` values
+        :type: feature
+        :pr: 3205
+
+        Use memoized ``request_class`` and ``response_class`` values
+
+    .. change:: Enable codegen backend by default
+        :type: feature
+        :pr: 3215
+
+        Enable the codegen backend for DTOs introduced in https://github.com/litestar-org/litestar/pull/2388 by default.
+
+    .. change:: Added precedence of CLI parameters over envs
+        :type: feature
+        :pr: 3190
+        :issue: 3188
+
+        - Added precedence of CLI parameters over envs
+
+    .. change:: Only print when terminal is ``TTY`` enabled
+        :type: feature
+        :pr: 3219
+
+        This PR sets the ``LITESTAR_QUIET_CONSOLE`` and ``LITESTAR_APP_NAME`` in the autodiscovery function.
```

Comment:
Nit but "this" is not a PR anymore so this wording doesn't really make sense here :)

Reply:
fixed 99ed5caf9008508a1883c467c9a2060ed1c8c4bb